feat: introduce WAL archive volume (#513)
cloudnative-pg/cloudnative-pg: docs/src/failure_modes.md

kubectl delete -n [namespace] pvc/[cluster-name]-[serial] pod/[cluster-name]-[serial]
```

!!! Note

>     In case you have instanciated a dedicated WAL volume it will also have to be deleted during this process.

Reviewer: ```suggestion
    In case you have instantiated a dedicated WAL volume it will also have to be deleted during this process.
```

Reviewer 2: Fixed

>     In case you have instantiated a dedicated WAL volume it will also have to be deleted during this process.

Reviewer 3: ```suggestion
    If you have specified a dedicated WAL volume, it will also have to be deleted during this process.
```
